h/traversal/roots.py

```diff
@@ -140,6 +140,13 @@ def __getitem__(self, client_id):
         return client
 
 
+class BulkAPIRoot(RootFactory):
+    """Root factory for the Bulk API."""
+
+    # Currently only LMS uses this end-point
```

Comment:
```suggestion
```

I think that's explicitly clear from the ACL list itself

Reply:
I don't think that's clear to people who are less familiar with H principals and authentication.